### DISCUSSION TITLE
Run in VS Code

### DISCUSSION THREAD

**Discussion by asemandeveloper** [langgenius/dify#4846] [Suggestion]
Hello to all
How to run the project in VS Code on Windows 11?
I want to run the project locally and develop it on my computer.
Please guide step by step.

**Comment by patryk20120** [comment]
What do you mean by `run in VS Code`? VS Code is an editor.

You can run the project using docker, please follow the steps https://docs.dify.ai/getting-started/install-self-hosted/docker-compose

**Reply by asemandeveloper** [reply]
My computer's operating system is Windows 11 and I have chosen VS Code for coding. I have made my own changes in the codes of the web folder. How do I ru  and implement the project?

**Reply by crazywoola** [reply]
This could help https://docs.dify.ai/getting-started/install-self-hosted/local-source-code
